.env.dev

```diff
@@ -25,3 +25,8 @@ export DATABASE_SSL=false
 export DATABASE_URL="postgres://postgres:root@localhost:5434/dev_prello"
 
 export TEST_DATABASE_URL="postgres://postgres:root@localhost:5434/test_prello"
+
+export SMTP_USER=""
```

Comment:
SMTP ?

Reply:
https://fr.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol

Reply:
It is the protocol used by the lib.